adapter/internal/oasparser/config_generator.go

```diff
@@ -127,17 +127,18 @@ func GetEnforcerAPI(mgwSwagger model.MgwSwagger, lifeCycleState string) *api.Api
 	}
 
 	return &api.Api{
-		Id:                 mgwSwagger.GetID(),
-		Title:              mgwSwagger.GetTitle(),
-		Description:        mgwSwagger.GetDescription(),
-		BasePath:           mgwSwagger.GetXWso2Basepath(),
-		Version:            mgwSwagger.GetVersion(),
-		ProductionUrls:     prodUrls,
-		SandboxUrls:        sandUrls,
-		Resources:          resources,
-		ApiLifeCycleState:  lifeCycleState,
-		Tier:               mgwSwagger.GetXThrottlingTier(),
-		SecurityScheme:     mgwSwagger.GetSetSecurityScheme(),
+		Id:                mgwSwagger.GetID(),
+		Title:             mgwSwagger.GetTitle(),
+		Description:       mgwSwagger.GetDescription(),
+		BasePath:          mgwSwagger.GetXWso2Basepath(),
+		Version:           mgwSwagger.GetVersion(),
+		ProductionUrls:    prodUrls,
+		SandboxUrls:       sandUrls,
+		Resources:         resources,
+		ApiLifeCycleState: lifeCycleState,
+		Tier:              mgwSwagger.GetXThrottlingTier(),
+		SecurityScheme:    mgwSwagger.GetSetSecurityScheme(),
+		AuthType:          mgwSwagger.GetXAuthType(),
```

Comment:
We need to have this in operation level , not only in the API level

Reply:
We have thin in operational level as well. Please refer GetEnforcerAPIOperation method.